src/bindings/python/src/pyopenvino/core/remote_context.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include <openvino/core/any.hpp>
+#include <openvino/core/type/element_type.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
+#include <openvino/runtime/intel_gpu/remote_properties.hpp>
+#include <openvino/runtime/remote_context.hpp>
+#include <openvino/runtime/tensor.hpp>
+
+#include "openvino/core/except.hpp"
+#include "pyopenvino/core/remote_tensor.hpp"
+
+#ifdef PY_ENABLE_GPU
+#ifndef _WIN32
+#    include <va/va.h>
+#endif  // _WIN32
+#endif  // PY_ENABLE_GPU
+
+namespace py = pybind11;
+
+class RemoteContextWrapper {
+public:
+    RemoteContextWrapper() {}
```

Comment:
what is the purpose of these wrappers?

Reply:
To provide similar inheritance scheme to one that GPU plugin is having, without actually including derived classes as they strongly rely on CL/VA headers.